src/backend/dx11/src/lib.rs

```diff
@@ -59,19 +59,18 @@ impl Instance {
     #[doc(hidden)]
     pub fn create() -> Self {
         // Create DXGI factory
-        let mut dxgi_factory = ComPtr::<winapi::IDXGIFactory2>::new(ptr::null_mut());
-
+        let mut dxgi_factory: *mut winapi::IDXGIFactory2 = ptr::null_mut();
         let hr = unsafe {
             dxgi::CreateDXGIFactory1(
                 &dxguid::IID_IDXGIFactory2,
-                dxgi_factory.as_mut() as *mut *mut _ as *mut *mut c_void)
```

Comment:
could you add `as_mut` method in the fork/upsteam? It seems quite useful on multiple occasions in our code